src/routes/Profile.jsx

```diff
@@ -0,0 +1,78 @@
+// React Imports
+import React, { useEffect, useState } from 'react';
+import { useLocation } from 'react-router-dom';
+// Inrupt Imports
+import { useSession } from '@inrupt/solid-ui-react';
+// Material UI Imports
+import Box from '@mui/material/Box';
+import Link from '@mui/material/Link';
+import Typography from '@mui/material/Typography';
+// Utility Imports
+import { fetchProfileInfo } from '../model-helpers';
+// Component Inputs
+import ProfileInputField from '../components/Profile/ProfileInputField';
+
+/**
+ * Profile Page - Page that displays the user's profile card information and
+ * allow users to edit/update them on PASS
+ *
+ * @memberof Pages
+ * @name Profile
+ * @returns {React.JSX.Element} The Profile Page
+ */
+const Profile = () => {
+  const location = useLocation();
+  const { session } = useSession();
+
+  localStorage.setItem('restorePath', location.pathname);
+
+  const [profileName, setProfileName] = useState(null);
+  const [organization, setOrganization] = useState(null);
+
+  const fetchProfileData = async () => {
+    const profileObject = await fetchProfileInfo(session);
+
+    setProfileName(profileObject.profileInfo.profileName);
+    setOrganization(profileObject.profileInfo.organization);
+  };
+
+  useEffect(() => {
```

Comment:
I'm trying to understand this `useEffect` better. 

If this is for first render, we could just have a `fetchProfileData()` without the `useEffect`.
If we are watching to see if the `profileName` is null, I don't think we should allow the user to be able to make it null or empty.

Reply:
Think we can make the dependency array empty so that it'll only run once when rendered. It was originally intended for just that.